src/buza/forms.py

```diff
@@ -3,11 +3,6 @@
 from buza.models import User
 
 
-class LoginForm(forms.Form):
```

Comment:
does the auth login not use an email by default?

Reply:
The default `django.contrib.auth.forms.AuthenticationForm` uses username + password authentication.